core/runtime/jsrealm.rs

```diff
@@ -62,10 +62,10 @@ pub(crate) struct ContextState<OpDriverImpl: OpDriver = DefaultOpDriver> {
     RefCell<Option<Rc<v8::Global<v8::Function>>>>,
   pub(crate) unrefed_ops:
     RefCell<HashSet<i32, BuildHasherDefault<IdentityHasher>>>,
-  pub(crate) pending_ops: OpDriverImpl,
+  pub(crate) pending_ops: Rc<OpDriverImpl>,
```

Comment:
I think this is fine - instead of cloning `Rc<ContextState>` for each op, we clone `OpDriver` instead.